Name: Just Charge By Default
Version: 3.0.2-1
Author: Jason Robitaille
Description: On plugging in your device, it'll automatically go into charging mode with notification if you want to change to a different mode  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/StoragedAlerts/StoragedAlerts.js
+++ /usr/lib/luna/system/luna-systemui/app/StoragedAlerts/StoragedAlerts.js
@@ -34,7 +34,7 @@
 				}
 			]
 		 },
-		 {kind: "ApplicationEvents", onWindowDeactivated:"handleWindowDeActivated"},
+		 {kind: "ApplicationEvents", onWindowActivated:"activated", onWindowDeactivated:"handleWindowDeActivated"},
 		 {kind: "NotificationButton", className:"enyo-notification-button", layoutKind:"HFlexLayout", 
 			 components:[
 		                   {flex:1, content: $L("USB Drive"), onclick: "enterMSM"},
@@ -59,6 +59,10 @@
 		this.tapOnButton = false;
 	},
 	
+	activated: function() {
+		this.charge();
+	},
+	
 	enterMSM: function(inSender) {
 		if(enyo.application.isDeviceLocked()) {
 			this.$.unlock.call({state:"undock"});
\ No newline at end of file
